ksqldb-execution/src/main/java/io/confluent/ksql/execution/codegen/SqlToJavaVisitor.java

```diff
@@ -764,11 +765,15 @@ private String visitBooleanComparisonExpression(final ComparisonExpression.Type
           ? process(node.getDefaultValue().get(), context).getLeft()
           : "null";
 
+      // ImmutableList.copyOf(Arrays.asList()) replaced ImmutableList.of() to avoid
```

Comment:
You might want to mention that it's the janino compiler that chokes on the combination of normal and variadic args (assuming we verified that as the cause).  This avoids that.

Reply:
I have linked the github issue because I thought that anyone who encounters this can go to that issue and check out the detailed discussions there